.github/workflows/layers_dispatch.yaml

```diff
@@ -0,0 +1,87 @@
+name: Deploy Layers
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # This will run the workflow daily at midnight
+
+env:
+  AWS_REGIONS: "us-east-1 us-east-2 us-west-1 us-west-2 ca-central-1 eu-central-1 eu-west-1 eu-west-2 eu-west-3 eu-north-1 ap-northeast-1 ap-northeast-2 ap-southeast-1 ap-southeast-2 ap-south-1 sa-east-1 af-south-1"
+  S3_BUCKET_NAME: "shelf-lambda-layers"
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Get Previous Tag Version
+        id: prev_tag_version
+        run: echo "::set-output name=tag::$(cat .tag_version || echo '')"
+
+      - name: Get Latest Release URL and Tag Version
+        id: latest_release
+        run: |
+          JSON_RESPONSE=$(curl -s https://api.github.com/repos/Sparticuz/chromium/releases/latest)
+          LATEST_RELEASE_URL=$(echo $JSON_RESPONSE | grep -Po '"zipball_url": "\K[^"]+')
```

Comment:
Shouldn't the `LATEST_RELEASE_URL` here point to `assets.0.browser_download_url` @iwaduarte ?
(https://api.github.com/repos/Sparticuz/chromium/releases/latest)

The unziped file downloaded through the `zipball_url` contains the _raw_ files of the [Sparticuz/chromium](https://github.com/Sparticuz/chromium/tree/master) repository. 
It doesn't contain a `node_modules` folder that could be _required_.

When I use `v35` of the layer I get the following error message.

```
 "Error: Cannot find module '@sparticuz/chromium'
Require stack:
- /var/task/browsershot.js
- /var/runtime/index.mjs. [TRACE] Runtime.ImportModuleError: Error: Cannot find module '@sparticuz/chromium' Require stack: - /var/task/browsershot.js".
```

When I create a new layer using `chromium-v116.0.0-layer.zip` from the latest [Sparticuz/chromium-release](https://github.com/Sparticuz/chromium/releases/tag/v116.0.0), my script finds the chromium instance.

Reply:
Oh yep. You are definitely right @stefanzweifel.  I have made a mistake. The download_url property is the url download not the zipball_url 😢.  

@vladgolubev

Reply:
@iwaduarte No problem. :) 
Thanks again for making this PR.

Reply:
I've published a fix, feel free to try now!

Reply:
Works flawlessly so far. Thank y'all! ♥️